build/archive.js

```diff
@@ -229,8 +229,13 @@ function exec(cmdLine, baton, callback) {
     }
 
     child_process.exec(cmdLine, function (err, stdout, stderr) {
-        var err = err || stderr;
-        err && baton ? error(err, baton) : callback((stdout || '').trim(), err);
+        if (!err) { err = stderr; }
+
+        if (err && baton) {
+            error(err, baton);
+        } else {
+            callback((stdout || '').trim(), err);
+        }
```

Comment:
Heh, I see you don't like the `||` and `&&` shorthand people use in JavaScript :). I guess I'm fine with the `&&` changes, as there are definitely strong opinions about how readable it is, but personally I think `err = err || stderr` is actually clearer than the alternative, particularly as it is a very common/standard JavaScript pattern.